lib/v2/nber/index.js

```diff
@@ -0,0 +1,49 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const path = require('path');
+const { art } = require('@/utils/render');
+const { parseDate } = require('@/utils/parse-date');
+
+async function getData(url) {
+    const response = await got(url).json();
+    return response.results;
+}
+
+module.exports = async (ctx) => {
+    const url = 'https://www.nber.org/api/v1/working_page_listing/contentType/working_paper/_/_/search';
+    const baseUrl = 'https://www.nber.org';
+    const data = ctx.cache.tryGet(url, async () => {
+        const response = await getData(url);
+        return response;
+    });
```

Comment:
`tryGet` is an async function and you need to `await` for it during assignment
```js
const config = require('@/config').value;
const data = await ctx.cache.tryGet(url, () => getData(url), config.cache.routeExpire, false);
```

Reply:
Can I just add `await` in front of `ctx.cache.tryGet` in line 15?

Reply:
The code works but it won't receive updated articles.
docs: https://docs.rsshub.app/joinus/quick-start.html#ti-jiao-xin-de-rsshub-gui-ze-bian-xie-jiao-ben-shi-yong-huan-cun
The cache will renew its expiry by default if it's a cache-hit situation. In other words, the list from the API won't get updated and you will always see the same list of articles.

Reply:
So we shouldn't use cache here?
Does cache-hit depend on the url passed in?

Reply:
Use cache but tells `tryGet` not to renew its expiry. That's why the 3rd and 4th parameters are needed. Thus,
```js
const config = require('@/config').value;
const data = await ctx.cache.tryGet(url, () => getData(url), config.cache.routeExpire, false);
```

Reply:
`getData()` is an async function, should I wrote like this?
```javascript
const data = await ctx.cache.tryGet(url, async () => {
    const response = await getData(url);
    return response;
}, config.cache.routeExpire, false);
```
ESlint says [no-await-return](https://eslint.org/docs/latest/rules/no-return-await)

Reply:
Either 
```js
const data = await ctx.cache.tryGet(url, () => getData(url), config.cache.routeExpire, false);
```
or
```js
    const data = await ctx.cache.tryGet(
        url,
        async () => {
            const response = await getData(url);
            return response;
        },
        config.cache.routeExpire,
        false
    );
```

ESLint will only triggers if you do
```js
    const data = await ctx.cache.tryGet(
        url,
        async () => {
            return await getData(url);
        },
        config.cache.routeExpire,
        false
    );
```

Reply:
Yeah.
I mean, will it make any differences if we do not add `async` and `await` to `getData()`

Reply:
No big differences when using in async function(`tryGet`). 

If you're interested in the technical details you can visit ESLint's [further reading](https://eslint.org/docs/latest/rules/no-return-await#further-reading).